bin/plugin/commands/performance.js

```diff
@@ -147,15 +133,12 @@ function curateResults( testSuite, results ) {
 		testSuite === 'front-end-classic-theme' ||
 		testSuite === 'front-end-block-theme'
 	) {
+		const timeToFirstByte = median( results.timeToFirstByte );
+		const largestContentfulPaint = median( results.largestContentfulPaint );
 		return {
-			timeToFirstByteMedian: median( results.timeToFirstByte ),
-			timeToFirstByteP75: percentile75( results.timeToFirstByte ),
-			largestContentfulPaintMedian: median(
-				results.largestContentfulPaint
-			),
-			largestContentfulPaintP75: percentile75(
-				results.largestContentfulPaint
-			),
+			timeToFirstByte,
+			largestContentfulPaint,
+			lcpMinusTtfb: largestContentfulPaint - timeToFirstByte,
```

Comment:
I see this is only computed here, meaning it won't be part of the logged results when you run the tests manually using `npm run test:performance ...` should we make the computation in the spec files directly instead and add a log to the performance logger as well?

Reply:
:+1: